ee/fe/modules/500-openvpn/templates/openvpn/statefullset.yaml

```diff
@@ -128,18 +162,31 @@ spec:
             {{- if hasKey .Values.openvpn "inlet" }}
               {{- if eq .Values.openvpn.inlet "LoadBalancer" }}
             --ovpn.server.behindLB
+                {{- if .Values.openvpn.udpEnabled }}
+                --ovpn.service="openvpn-external-udp"
+                {{- end }}
+                {{- if .Values.openvpn.tcpEnabled }}
             --ovpn.service="openvpn-external"
```

Comment:
```suggestion
            --ovpn.service="openvpn-external-tcp"
```
And everywhere.

Reply:
It is forbidden in kubernetes to rename objects without recreating them. We cannot recreate the service, because the load balancer will be reordered.